django_ipam/__init__.py

```diff
@@ -0,0 +1,17 @@
+VERSION = (0, 1, 0, 'alpha')
+__version__ = VERSION  # alias
+
+
+def get_version():
+    version = '%s.%s' % (VERSION[0], VERSION[1])
+    if VERSION[2]:
+        version = '%s.%s' % (version, VERSION[2])
+    if VERSION[3:] == ('alpha', 0):
+        version = '%s pre-alpha' % version
+    else:
+        if VERSION[3] != 'final':
+            version = '%s %s' % (version, VERSION[3])
+    return version
+
+
+default_app_config = 'django_ipam.apps.DjangoIPAMConfig'
```

Comment:
rename to `django_ipam.apps.DjangoIpamConfig` (as well as the class it points to) for better camelCase adherence